BasicSample/Assets/ARAnchor/Scripts/AnchorsSample.cs

```diff
@@ -207,7 +213,11 @@ public void AddAnchor(Pose pose)
         {
 #if AR_FOUNDATION_4_1_1_OR_LATER
             Debug.Log($"Instantiating new GameObject containing an ARAnchor");
-            Instantiate(m_arAnchorManager.anchorPrefab, pose.position, pose.rotation);
+
+            // When instantiating a trackable gameobject, it should be a child of the ARSessionOrigin.trackablesParent GameObject.
+            // If the GameObject is not a child of this trackablesParent, it may not be cleaned up properly in some scenarios,
+            // such as scene changes. This is especially important for applications composed of additive scenes, like this one.
+            Instantiate(m_arAnchorManager.anchorPrefab, pose.position, pose.rotation, m_arSessionOrigin.trackablesParent);
```

Comment:
In the Main Test project, we stopped using this instantiate method as it is buggy. Do we want to keep using this in the sample project? 
https://github.com/microsoft/Unity-MR-Plugin/blob/main/TestProjects/Main/Assets/Scripts/AnchorsSample.cs#L175

Reply:
fair enough. The issue we opened with unity is still under discussion. We can change accordingly later.